Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
+
+RUN apk update
+RUN apk --no-cache add curl
+RUN apk add jq
+
+COPY entrypoint.sh /entrypoint.sh
+
+ENTRYPOINT ["sh", "/entrypoint.sh"]
```

Comment:
this Dockerfile will be built every time we run the action so maybe find an image already providing these tools or have two dockerfiles (one to run, one to build and that is published everytime we commit new stuff there)

RUN instructions should be merged